docs/docusaurus/docs/guides/connecting_to_your_data/datasource_configuration/components/_section_create_a_new_datasource_configuration.mdx

```diff
@@ -1,21 +1,21 @@
-A new Datasource can be configured in Python as a dictionary with a specific set of keys.  We will build our Datasource configuration from scratch in this guide, although you can just as easily modify an existing one.
+A new Data Source can be configured in Python as a dictionary with a specific set of keys.  We will build our Data Source configuration from scratch in this guide, although you can just as easily modify an existing one.
 
 To start, create an empty dictionary.  You will be populating it with keys as you go forward.
 
-At this point, the configuration for your Datasource is merely:
+At this point, the configuration for your Data Source is merely:
 
 ```python name="datasource_configuration_empty_dictionary"
 ```
 
-However, from this humble beginning you will be able to build a full Datasource configuration.
+However, from this humble beginning you will be able to build a full Data Source configuration.
 
-#### The keys needed for your Datasource configuration
+#### The keys needed for your Data Source configuration
 
-At the top level, your Datasource's configuration will need the following keys:
-- `name`: The name of the Datasource, which will be used to reference the datasource in Batch Requests.
-- `class_name`: The name of the Python class instantiated by the Datasource.  Typically, this will be the `Datasource` class.
+At the top level, your Data Source's configuration will need the following keys:
+- `name`: The name of the Data Source, which will be used to reference the datasource in Batch Requests.
+- `class_name`: The name of the Python class instantiated by the Data Source.  Typically, this will be the `Data Source` class.
```

Comment:
```suggestion
- `class_name`: The name of the Python class instantiated by the Data Source.  Typically, this will be the `Datasource` class.
```
If it's in code backticks, the proper class name is `Datasource`.